ros2topic/ros2topic/verb/pub.py

```diff
@@ -172,6 +177,14 @@ def publisher(
 
     pub = node.create_publisher(msg_module, topic_name, qos_profile)
 
+    times_since_last_log = 0
+    while pub.get_subscription_count() < wait_matching_subscriptions:
+        # Print a message reporting we're waiting each 1s, check condition each 100ms.
+        if not times_since_last_log:
+            print(f'Waiting for {wait_matching_subscriptions} matching subscription(s)...')
```

Comment:
I'll suggest:

```suggestion
            print(f'Waiting for at least {wait_matching_subscriptions} matching subscription(s)...')
```

Reply:
```suggestion
            print(
                f'Waiting for at least {wait_matching_subscriptions} matching subscription(s)...')
```

Reply:
Fixed in https://github.com/ros2/ros2cli/pull/642/commits/3d59b36971615b8cab6f733578023b182b833864